engo_js.go

```diff
@@ -200,6 +207,7 @@ func rafPolyfill() {
 // RunIteration runs one iteration per frame
 func RunIteration() {
 	Time.Tick()
+	Input.clear()
 	Input.update()
 	currentWorld.Update(Time.Delta())
```

Comment:
@Sendoushi Please explain what you mean by this in detail somewhere else (issue, gitter) because you keep mentioning it and I have no idea what you're saying.